### PR TITLE
Finished bootcamp

### PR DESCRIPTION
**Pull request by Azhng**

### CODE REVIEW

File: modules/core/CMakeLists.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
 include_directories(include)
+set(SOURCE_FILES include/merge_sort.h)
```
Comment (benjaminwinger) [inline modules/core/CMakeLists.txt]: If you're not using this variable, you should probably remove it

File: modules/core/test/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,15 @@
+include_directories(${Core_INCLUDE_DIR})
```
Comment (benjaminwinger) [inline modules/core/test/CMakeLists.txt]: Core_INCLUDE_DIR isn't set here, but it looks like the reason this still works is that CMake actually inherits include directories from its parent CMakeLists as long as you call include_directories before adding the subdirectory.

**Review by benjaminwinger — CHANGES_REQUESTED** [review]
A couple housekeeping issues, but other than that it looks fine.

**Review by Azhng — COMMENTED** [review]
Removed redundant code

**Review by benjaminwinger — APPROVED** [review]

**Pull request merged**